src/styles/main.scss

```diff
@@ -78,97 +157,464 @@
+  margin: auto;
+  #newsletter-email {
+    box-shadow: 0 5px 35px $color-c6;
+  }
+  .form-control {
+    border-radius: 30px;
+    border-top-right-radius: 30px !important;
+    border-bottom-right-radius: 30px !important;
+    line-height: 26px;
+    border: none;
+    background-color: $color-c5;
+    color: $color-c4;
+    font-weight: $font-weight-s2;
+    letter-spacing: $letter-spacing-t1;
+    padding-left: 30px;
+  }
+  .input-group-append {
+    position:  absolute;
+    right:  3px;
+    margin-top: 2px;
+    .input-group-text {
+      border-radius: 30px;
+      padding: 5px 20px;
+      background-color: $color-c1;
+      text-transform: uppercase;
+      letter-spacing: $letter-spacing-t3;
+      font-weight: $font-weight-s3;
+      border: none;
+      color: $color-c5;
+      z-index: 4;
+      &:hover {
+        background-color: $color-c3;
+      }
+    }
+  }
+}
 
-#carouselForm {
-  padding-top: 30px;
-  height: 100%;
-  display: flex;
-  flex-direction: column;
-  .carousel-inner {
-    display: flex;
-    flex:1;
+.stop {
+  background-color: $color-c4;
+  color: $color-c5;
+  text-align: left;
+  padding: 40px 40px 200px 40px;
+  h3 {
+    color: $color-c3;
+    font-weight: $font-weight-s3;
+    text-transform: uppercase;
+    font-size: 1em;
+    img {
+      max-width: 30px;
+      margin: 10px;
+    }
   }
-  .carousel-indicators li {
-    text-indent: initial;
-    height: auto;
-    background-color: transparent;
-    border: 2px dotted;
-    text-align: center;
-    padding: 0px 23px 0px 11px;
-    border-radius: 50%;
-    font-size: 1.5em;
-    &.active {
-      border-style: solid;
+
+  h4 {
+    margin-bottom: 0px;
+    &#typewriter {
+      color: $color-c3;
+      letter-spacing: $letter-spacing-t1;
+      margin-bottom: 20px;
     }
-    &::before, &::after {
-      display: none;
+  }
+}
+
+.comminity-section {
+  position: relative;
+  padding-bottom: 250px;
+  .community {
+    padding: 40px;
+    background-color: $color-c5;
+    color: $color-c4;
+    letter-spacing: $letter-spacing-t1;
+    text-align: left;
+    .community-users {
+      max-width: 100px;
+    }
+    .description {
+      color: $color-c1;
+      font-weight: $font-weight-s2;
+      a {
+        color: $color-c1;
+        &:hover {
+          text-transform: none;
+        }
+      }
+    }
+  }
+
+  .values {
+    position: absolute;
+    top: 250px;
+    width: 100%;
+    .thumbnail {
+      margin: 80px 30px;
+      .caption {
+        p {
+          text-align: center;
+        }
+        h3 {
+          font-size: 1.5em;
+          font-weight: $font-weight-s3
+        }
+      }
+      img {
+        height: 50px;
+        margin-bottom: 15px;
+      }
+    }
+    .universal {
+      background: linear-gradient(180deg, #FFFFFF 0%, #d7d7d7 100%);
+    }
+    .no-signup {
+      background: linear-gradient(180deg, $color-c3 0%, $color-c2 100%);
+      color: $color-c5;
+      box-shadow: 0px 10px 60px 0px #000000;
+    }
```

Comment:
I would add an index on middle card in order to add shadow on both left and right cards.

`z-index:1`